akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -767,6 +767,7 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         // tear down the upstream hub part if downstream lane fails
         // lanes are not completed with success by themselves so we don't have to care about onSuccess
         completed.failed.foreach { reason ⇒ hubKillSwitch.abort(reason) }
+        completed.onComplete { x ⇒ println(s"# completed: $x") } // FIXME
 
```

Comment:
I tried one more debug:
```
        completedValues.zipWithIndex.map {
          case (c, i) ⇒
            c.onComplete { x ⇒ println(s"# completed lane: $i => $x") } // FIXME
        }
```

From that I can see that one of the lanes is completed with failure, but the aggregated Future is not completed. How is that possible. It's as simple as:
```
val completed = Future.sequence(completedValues).map(_ ⇒ Done)
```